manifests/filter/system_model.pp

```diff
@@ -137,6 +137,7 @@
   $rewrite_host_header = undef,
   $service_cluster     = undef,
   $tracing_header      = {},
+  $encoded_headers      = [],
```

Comment:
One Style nitpick here - please lineup the "=" with the rest of the attributes. 
Also please document the new attribute higher up in the file in the file following the style of documentation for the other attributes.

Reply:
@bjoshua documented the new attribute and fixed this lineup